src/main/java/com/board/tddboard/dto/BoardDto.java

```diff
@@ -0,0 +1,51 @@
+package com.board.tddboard.dto;
+
+import com.board.tddboard.domain.Board;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotEmpty;
+
+public class BoardDto {
+
+    @Getter @Setter
+    public static class CreateDto {
+
+        @NotEmpty(message = "제목이 존재하지 않습니다.")
```

Comment:
```suggestion
        @NotBlank(message = "제목이 존재하지 않습니다.")
```

`@NotEmpty` 로 검사하면 공백검사를 안해요 공백도 검사해야하지 않을까요?